README.md

```diff
@@ -140,15 +140,22 @@ you must use the `--config` option:
 
 ```
 cd ~/dirA
-twigcs --config ~/dirB/.twig_cs.dist # Will lint templates in ~/dirA with the config of ~/dirB
+twigcs --config ~/dirB/.twig_cs.dist.php # Will lint templates in ~/dirA with the config of ~/dirB
 ```
 
-By default, the files `.twig_cs` and `.twig_cs.dist` are looked up in your current working directory (CWD).
+By default, the files
+
+- `.twig_cs.php`
+- `.twig_cs`
+- `.twig_cs.dist.php`
+- `.twig_cs.dist`
```

Comment:
@yguedidi 

I have turned this into a list, making the order perhaps a bit more clear. What do you think?

Reply:
Indeed better!